src/main/kotlin/com/sourcegraph/cody/agent/protocol/ChatMessage.kt

```diff
@@ -0,0 +1,10 @@
+package com.sourcegraph.cody.agent.protocol
+
+data class ChatMessage(
+    override val speaker: Speaker,
+    override val text: String?,
+    val displayText: String? = null,
+    val contextFiles: List<ContextFile>? = null
```

Comment:
does it make sense to distinguish `contextFiles`: `null` and `emptyList`?

Reply:
Just mirroring how its defined in the cody repo. Im not sure how deserializing `null` works into a property with a default value e.g. if this was `val contextFiles: List<ContextFile> = emptyListOf()` :thinking: